models/models.go

```diff
@@ -0,0 +1,36 @@
+package models
+
+type AssetType int
+
+type Chart struct {
+	Title string
+	XAxis string
+	YAxis string
+	Data  map[string]float64
+}
+
+type Insight struct {
+	Text string
+}
+
+type Audience struct {
+	Gender             string
+	BirthCountry       string
+	AgeGroup           string
+	HoursOnSocialMedia int
+	PurchasesLastMonth int
+}
+
+type Asset struct {
+	ID          string
+	Type        AssetType
+	Description string
+	Chart       *Chart
+	Insight     *Insight
+	Audience    *Audience
+}
```

Comment:
I left this question last, because I must admin it seemed as the most difficult to answer. I did a lot of searching and reading regarding structs and whether I shuld have used values or pointers and to be really honest, I haven’t decided yet… So my answer is that taking into consideration that the values of Chart, Insight and Audience could not be change, (the user only changes the description), the use of pointers here is useless.